client/src/views/v2/SensorStatusView.js

```diff
@@ -12,6 +12,9 @@ import { useStatus } from 'api/v2/sensors';
 export default function SensorStatusView() {
   const sensorStatus = useStatus();
 
+  sensorStatus.push(useCameraStatus("primary"));
+  sensorStatus.push(useCameraStatus("secondary"));
```

Comment:
Not sure how I feel about this being on the sensor status page. Perhaps this should become more of a system status page, and I was thinking that's what we'd do for V3, but for now, would this information be better presented somewhere on the camera page? Open to discussion. I'm not sure exactly *where* it should go on the camera page - its awkward having to present two very similar camera statuses.

Reply:
Done.
8a14836

I think we can leave this here for now, and when we do the design for V3 there will be a page for system status.

![image](https://user-images.githubusercontent.com/50545432/92920092-117d5980-f475-11ea-81d1-25d78676ebbb.png)
Thoughts on formatting?

Reply:
Happy with that, for now. I think this page has quickly become rather cluttered recently, so it's probably in need of a redesign sometime in the future once we've done the important new V3 stuff. For now, the functionality is there so I'm happy!